src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -814,5 +858,306 @@ <h3>What happens next</h3>
+      </td>
+    </tr>
+  </table>
+</header>
+<table style="width: 80%" class="new-dq">
+  <thead>
+  <tr>
+    <th colspan="2">
+      <h2 class="heading-medium" id="name">
+        <span>A</span> Your details - Defendant
+      </h2>
+    </th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td>
+      <div>
+        <strong>Name:</strong></div>
+    </td>
+    <td>
+      <div>{{ defendant.fullName }}</div>
+    </td>
+  </tr>
+  <tr>
+  </tr>
+  <tr>
+    <td>
+      Address:
+    </td>
+    <td>
+      <div>
+        {{ defendant.address.line1 }}
+        <br/>
+        {% if defendant.address.line2 is defined and defendant.address.line2 is not empty %}
+        {{ defendant.address.line2 }}
+        <br/>
+        {% endif %}
+        {% if defendant.address.line3 is defined and defendant.address.line3 is not empty %}
+        {{ defendant.address.line3 }}
+        <br/>
+        {% endif %}
+        {{ defendant.address.city }}
+        <br/>
+        {{ defendant.address.postcode }}
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <div>
+        Date of birth:
+      </div>
+    </td>
+    <td>
+      <div>
+        {{ defendant.dateOfBirth }}
+      </div>
+    </td>
+  </tr>
+  <tr></tr>
+  <tr>
+    <td>
+      <div>
+        Telephone:
+      </div>
+    </td>
+    <td>
+      <div>
+        {% if defendant.mobilePhone is defined and defendant.mobilePhone is not empty %}
+        {{ defendant.mobilePhone }}
+        {% endif %}
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <div>
+        Email:
+      </div>
+    </td>
+    <td>
+      <div>{{ defendant.email }}</div>
+    </td>
+  </tr>
+  </tbody>
+</table>
+
+<table style="width: 80%" class="new-dq">
+  <thead>
+  <tr>
+    <th colspan="2">
+      <h2 class="heading-medium">
+        <span>B</span> About the hearing
+      </h2>
+    </th>
+  </tr>
+  </thead>
+  <tbody>
+
+  <tr>
+    <td>
+      <div>
+        Support required for a hearing
+      </div>
+    </td>
+    <td>
+      <div>
+        {% for support in hearingContent.supportRequired %}
+        {{ support }}<br/>
+        {% endfor %}
+      </div>
+    </td>
+  </tr>
+
+  <tr>
+    <td><b>Hearing location</b></td>
+  </tr>
+  <tr>
+    <td>
+      Preferred hearing centre
+    </td>
+    <td>
+      <div>{{ hearingContent.hearingLocation }}</div>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <div>
+        Briefly explain your reasons
+      </div>
+    </td>
+    <td>
+      <div>
+        {{ hearingContent.locationReason }}
+      </div>
+    </td>
+  </tr>
+
+  <tr>
+    <td><b>Expert evidence</b></td>
+  </tr>
+  <tr>
+    <td>
+      Have you already got a report written by an expert?
+    </td>
+    <td>
+      <div>
+        {{ hearingContent.hasExpertReport }}
+      </div>
+    </td>
+  </tr>
+  {% for report in hearingContent.expertReports %}
+  <tr>
+    <td>
+      <div>
+        Report {{ loop.index }}
```

Comment:
Are you sure you don't need `(loop.index +1)` ? 
Also if there is only one item in collection, normally we don't display  the number. i.e. `Report` instead of `Report 1` when size is one of the expertReports collection. Please check with P.O.

Reply:
Hi @sabahirfan I have to push the changes. Currently merging backend requirement for downloading document as well. Soon will push the change

Reply:
Great thanks 👍